src/protocol/protocol.ts

```diff
@@ -33,6 +33,16 @@ export namespace Protocol {
         params: ServerAttributes;
     }
 
+    export interface Attributes {
```

Comment:
both these interfaces are already declared in this file, no need to add them anew